src/components/citiesIndexes.js

```diff
@@ -0,0 +1,360 @@
+const citiesIndexes = [
+  {
+    cd: 'AD',
+    start: 0,
+    end: 9,
+  },
+  {
+    cd: 'AE',
+    start: 10,
+    end: 23,
+  },
+  {
+    cd: 'AF',
+    start: 24,
+    end: 334,
+  },
+  {
+    cd: 'AG',
+    start: 335,
+    end: 340,
+  },
+  {
```

Comment:
- [ ] instead of using an object to set cities in your select city, please use the props to make it.